_source/api/logzio-public-api.yml

```diff
@@ -4813,3 +4932,369 @@ definitions:
+
+
+          The minimum recommended time frame is 5 minutes, as anything shorter will be less reliable and unnecessarily resource-heavy.
+
+
+          The maximum recommended time frame is 1440 minutes (24 hours). The alert runs on the index from today and yesterday (in UTC) and the maximum time frame increases throughout the day, reaching 48 hours exactly before midnight UTC.
+        example: 20
+      subComponents:
+        type: array
+        description: Sets the search criteria using a search query, filters, group by aggregations, accounts to search, and trigger conditions.
+        items:
+          $ref: '#/definitions/SubAlert'
+      correlations:
+        $ref: '#/definitions/SubAlertCorrelation'
+      enabled:
+        type: boolean
+        description: If `true`, the alert is enabled and active.
+  AlertTrigger:
+    type: object
+    description: Sets an additional triggering threshold and severity tab to label the event when the alert triggers.
+    properties:
+      operator:
+        type: string
+        enum:
+          - LESS_THAN
+          - GREATER_THAN
+          - LESS_THAN_OR_EQUALS
+          - GREATER_THAN_OR_EQUALS
+          - EQUALS
+          - NOT_EQUALS
+        example: GREATER_THAN_OR_EQUALS
+        description: Specifies the operator for evaluating the results.
+      severityThresholdTiers:
+        type: object
+        description: >-
+          Sets a severity label per trigger threshold as a key:value pair.
+          
+          
+          If using more than one sub-component, only 1 severityThresholdTiers is allowed. Otherwise, 1 per `enum` are allowed (for a total of 5 thresholds of increasing severities).
+
+
+          Increasing severity must adhere to the logic of the operator.
+        enum:
+          - INFO
+          - LOW
+          - MEDIUM
+          - HIGH
+          - SEVERE
+        additionalProperties:
+        default:
+          MEDIUM: 10.0
+        example:
+          MEDIUM: 10.0
+          HIGH: 100.0
+          SEVERE: 300.0
+
+  BoolFilter:
+    type: object
+    description: Apply `must` and `must_not` filters to the monitoring alert. Filters are more efficient compared to a query, so it's recommended to opt for a filter over a query, where possible. See [Elasticsearch Bool-Query](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/query-dsl-bool-query.html) for more detail.
+    properties:
+      bool:
+        $ref: '#/definitions/FilterLists'
+  ColumnConfig:
+    type: object
+    description: Customize the alert output to be sent out in notifications when the alert triggers.
+    properties:
+      fieldName:
+        type: string
+        description: Specify the fields to be included in the notification.
+      regex:
+        type: string
+        description: Trims the data using regex filters. [Learn more](https://docs.logz.io/user-guide/alerts/regex-filters.html)
+      sort:
```

Comment:
an exception is thrown if the user selects an "analyzed field" to be the fieldName AND sort to be either of the enum values. in other words - you cant sort by an analyzed field.

Reply:
Updated the description.